build.rs

```diff
@@ -65,14 +66,23 @@ fn main() {
 /// Start the recursion from the top level of the frontend's dist folder
 fn acquire_assets() -> Result<Vec<Asset>> {
     let mut output: Vec<Asset> = Vec::new();
-
-    for entry in WalkDir::new(DIST).into_iter() {
-        let entry = entry?;
-        if entry.metadata()?.is_file() {
-            output.push(Asset {
-                // If the directory isn't valid this wouldn't have worked.
-                path: String::from(entry.path().to_str().unwrap()).replace("\\", "/"),
-            });
+    let whitelist = vec![
+        "assets/*",
```

Comment:
This particular pattern will not. If we write it as `assets/**` it will include subfolders. The glob pattern syntax is [here](https://doc.rust-lang.org/glob/glob/struct.Pattern.html).